ugs-pendant/src/main/webapp/proxy.conf.json

```diff
@@ -0,0 +1,8 @@
+{
```

Comment:
When developing on the web client, this setting will allow to connect to a running API